components/Feed/FeedGrid/__tests__/FeedGrid.test.tsx

```
@@ -0,0 +1,769 @@
+
+jest.mock("masonic", () => ({
+  Masonry: ({
+    items,
+    render,
+  }: {
+    items: unknown[];
+    render: (props: { data: unknown }) => React.ReactNode;
+  }) => (
+    <div
+      data-testid="masonry"
+      role="main"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<main>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)